lib/experimental-i18n-theme.json

```diff
@@ -14,5 +14,8 @@
 				"gradients": [ "name" ]
 			}
 		}
+	},
+	"pageTemplates": {
```

Comment:
:wave: @swissspidy what would you think of declaring the field this way? For context, this is [the field in use in a theme](https://github.com/WordPress/theme-experiments/blob/master/tt1-blocks/experimental-theme.json#L308).

Reply:
I suppose it works, but I'll know for sure once I get around to update WP-CLI to use this file.

Reply:
Aside: I don't don't agree  with the naming chosen in #27778 / #28700. WordPress calls them `post templates` internally (see `\WP_Theme::get_post_templates()`), not page templates.

This should really be `postTemplates`.

Reply:
ok, thanks, going to implement this as it is.

As for the name: I don't have an opinion other than following the existing name. Can change that later if there's an update to it.

Reply:
As of #28830, it's now `customTemplates`.